lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -732,8 +732,11 @@ def get(self, request, *args, **kwargs):
             cilset = CourseImportLog.objects.filter(course_id=course_id).order_by('-created')
             log.debug('cilset length={0}'.format(len(cilset)))
         mdb.disconnect()
-        context = {'cilset': cilset,
-                   'course_id': course_id.to_deprecated_string() if course_id else None,
-                   'error_msg': error_msg}
+        context = {
+            'cilset': cilset,
+            'course_id': course_id.to_deprecated_string() if course_id else None,
+            'detailed_view': course_id is not None,
```

Comment:
I agree with @sarina , this doesn't seem to be adding much over just checking if it is set or not in the template.